src/lir/expr/expression.rs

```diff
@@ -783,6 +836,12 @@ impl Compile for Expr {
                     Self::Mul(_, _) => StandardOp::Mul { src, dst },
                     Self::Div(_, _) => StandardOp::Div { src, dst },
                     Self::Rem(_, _) => StandardOp::Rem { src, dst },
+                    Self::Lt(_, _) => StandardOp::IsLess { a: src, b: dst.clone(), dst },
+                    Self::Gt(_, _) => StandardOp::IsGreater { a: src, b: dst.clone(), dst },
+                    Self::Le(_, _)
+                    | Self::Ge(_, _)
+                    | Self::Eq(_, _)
+                    | Self::Neq(_, _) => StandardOp::CoreOp(CoreOp::Many(vec![])),
```

Comment:
Should put a comment here why this isn't "unreachable". It's because earlier the operator might be let through because it can't be ruled out that the arguments to the comparison might be integers.